frontend/src/pages/distributedWorkloads/global/GlobalDistributedWorkloads.tsx

```diff
@@ -33,31 +34,39 @@ const GlobalDistributedWorkloads: React.FC<GlobalDistributedWorkloadsProps> = ({
       />
     );
   }
-
-  if (activeTab.projectSelectorMode !== null) {
-    if (!namespace && (activeTab.projectSelectorMode === 'singleProjectOnly' || preferredProject)) {
-      // No namespace is in the path and either this tab requires one or we have a preferred one to go to
-      const redirectProject = preferredProject ?? projects[0];
-      return <Navigate to={getInvalidRedirectPath(redirectProject.metadata.name)} replace />;
-    }
-    if (namespace && !projects.find(byName(namespace))) {
-      // This tab allows or requires a namespace but was given an invalid one.
-      return (
-        <ApplicationsPage
-          {...{ title, description }}
-          loaded
-          empty
-          emptyStatePage={
-            <InvalidProject namespace={namespace} getRedirectPath={getInvalidRedirectPath} />
-          }
-        />
-      );
-    }
+  if (!namespace) {
+    // No namespace is in the path and either this tab requires one or we have a preferred one to go to
```

Comment:
This comment can probably be removed because "either this tab requires one or we have a preferred one to go to" doesn't apply anymore and "No namespace is in the path" is obvious